frontend/ui/data/onetime_migration.lua

```diff
@@ -355,5 +355,16 @@ if last_migration_date < 20220116 then
     end
 end
 
+-- Disable the jump marker on the Libra 2, to avoid the worst of epdc race issues...
+-- c.f., https://github.com/koreader/koreader/issues/8414
+if last_migration_date < 20220205 then
+    logger.info("Performing one-time migration for 20220205")
+
+    local Device = require("device")
+    if Device:isKobo() and Device.model == "Kobo_io" then
+        G_reader_settings:makeFalse("followed_link_marker")
+    end
```

Comment:
Is this worth an entry in onetime_migration.lua - assuming one day, we might solve the issue, and we'll have to add as much to remove that setting. (Or we can just replace the date, and makeFalse with delSetting?)
Can't we stick this in kobo/device.lua ? (Although the day we don't need it, we'll have to revert it in onetime_migration....)
Well, ok, just sharing these passing thoughts :)

Reply:
Yeah, I'm happier with a couple entries in here that fiddling with it in Device, where it would feel clunkier.